csrc/executor.h

```diff
@@ -398,6 +409,14 @@ class FusionExecutor : public NonCopyable {
       int64_t runtime_id,
       int64_t group_id);
 
+  //! Check if compilation was skipped (fusion segment marked for EE).
+  bool isCompilationSkipped() const {
+    if (fusion_ != nullptr) {
+      NVF_ERROR(!lowered_, "Expected GPU lowering to be skipped.");
+    }
```

Comment:
```suggestion
    NVF_ERROR(fusion_ == nullptr || !lowered_, "Expected GPU lowering to be skipped.");
```

nitpicking is all I do.